javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,10 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var nums = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'number') {
```

Comment:
This will catch all numbers, but there are more data types that aren't strings than just numbers. You really want to be saving values that _aren't a string_